wdl-lint/tests/lints/deprecated-placeholder-options-v1.1/source.errors

```diff
@@ -46,3 +46,11 @@ note[DeprecatedPlaceholderOption]: use of the deprecated `default` placeholder o
    │
    = fix: replace the `default` placeholder option with a call to the `select_first()` standard library function
 
+note[CommandSectionShellCheck]: `shellcheck` reported the following diagnostic
```

Comment:
I think this must have been caused by the random variable starting with two digits. I'll update so that it always starts with a letter.